packages/vscode-extension/src/envTree.ts

```diff
@@ -34,6 +39,7 @@ export async function registerEnvTreeHandler(): Promise<Result<Void, FxError>> {
           contextValue: "environment",
           icon: "symbol-folder",
           isCustom: false,
+          description: item === activeEnv ? "active" : "",
```

Comment:
The string seems visible to users, better move to strings.json

Reply:
Fixed.